src/mol-model-formats/structure/lammps-data.ts

```diff
@@ -32,23 +32,27 @@ async function getModels(mol: LammpsDataFile, ctx: RuntimeContext, unitsStyle: U
     const cz = new Float32Array(count);
     const model_num = new Int32Array(count);
 
+    const ll = bounds.length;
+    const lo = bounds.lower;
     let offset = 0;
     for (let j = 0; j < count; j++) {
         type_symbols[offset] = atoms.atomType.value(j).toString();
-        cx[offset] = atoms.x.value(j) * scale;
-        cy[offset] = atoms.y.value(j) * scale;
-        cz[offset] = atoms.z.value(j) * scale;
+        cx[offset] = ((atoms.x.value(j) - lo[0]) % ll[0] + lo[0]) * scale;
+        cy[offset] = ((atoms.y.value(j) - lo[1]) % ll[1] + lo[1]) * scale;
+        cz[offset] = ((atoms.z.value(j) - lo[2]) % ll[2] + lo[2]) * scale;
         id[offset] = atoms.atomId.value(j) - 1;
         model_num[offset] = 0;
         offset++;
     }
 
     const MOL = Column.ofConst('MOL', count, Column.Schema.str);
-    const asym_id = Column.ofLambda({
-        value: (row: number) => atoms.moleculeId.value(row).toString(),
-        rowCount: count,
-        schema: Column.Schema.str,
-    });
+    const asym_id = (count > 200000) ? Column.ofConst('A', count, Column.Schema.str)
```

Comment:
can you please make this a parameter like unitsStyle?

Reply:
you mean like a boolean flag, called for instance "asymId": true | false ?

Reply:
yes, make it a boolean parameter here. In CoordinatesFromLammpstraj & TrajectoryFromLammpsTrajData (src\mol-plugin-state\transforms\model.ts) you can do a param with 3 options: on, off, auto. For auto you can use a hardcoded value (for now, but we can make it easily configurable because it is a the plugin level)